nomenclature/code.py

```diff
@@ -209,3 +211,11 @@ class RegionCode(Code):
     """
 
     hierarchy: str = None
+    countries: List[str] = None
+
+    def check_iso3_codes(self) -> bool:
+        for country in self.countries:
+            if pycountry.countries.get(alpha_3=country) is None:
+                logging.warning(f"{country} is not a valid iso3 country code!")
```

Comment:
Better to keep a list all invalid countries in this check, and return a warning like:
```python
f"Region {name} has invalid ISO3 country codes: {invalid_iso3_codes}"
```

Reply:
That would be good to turn into a pydantic validator. Reason being is that it is then automatically called whenever you create an instance of `RegionCode`. For now it has to be called explicitly.